{{ cookiecutter.repo_name }}/Makefile

```diff
@@ -14,53 +14,26 @@ export
 # Import config variables
 include .cookiecutter/config
 
-# Ensure directory to track and log setup state exists
-$(shell mkdir -p .cookiecutter/state)
-
-.PHONY: test-setup
-## Test that everything has been setup
-test-setup:
-	@echo Testing configuration
-
-	@# Checking S3
-	@$(call execute_in_env, (test '${IN_PYTEST}' && echo 'In test-suite: Skipping S3 checks') ||\
-	 aws s3 ls s3://${BUCKET} > /dev/null 2>&1 || echo "ERROR: No bucket")
-
-	@# Github
-	@(test '${IN_PYTEST}' && echo 'In test-suite: Skipping Github checks') ||\
-	 git ls-remote > /dev/null 2>&1 || echo "ERROR: No Git remote"
-
-	@# Pre-commit valid
-	@$(call execute_in_env, pre-commit run -a)
-
-	@# Metaflow
-	@$(call execute_in_env, python .cookiecutter/scripts/check_metaflow_aws.py || echo "ERROR: Metaflow+AWS configuration")
-
-.PHONY: init
-## Fully initialise a project: install; setup github repo; setup S3 bucket
-init:  install .cookiecutter/state/setup-bucket .cookiecutter/state/setup-github
-	@echo SETUP COMPLETE
-
 .PHONY: install
 ## Install a project: create conda env; install local package; setup git hooks; setup metaflow+AWS
-install: .cookiecutter/state/conda-create .cookiecutter/state/setup-git .cookiecutter/state/setup-metaflow
+install: .cookiecutter/state/conda-create .cookiecutter/state/setup-git
```

Comment:
Just noticed removing this means you may want to edit line 11 of the README which still says:
```
- Run `make install` to configure the development environment:
  - Setup the conda environment
  - Configure pre-commit
  - Configure metaflow to use AWS
  ```